Tests/CachedSettingsManagerTest.php

```diff
@@ -109,20 +109,18 @@ public function testSet()
         $settingsManager->shouldReceive('set')->once()->with($name, $value, $owner);
 
         $cachedSettingsManager = $this->getMockBuilder(CachedSettingsManager::class)
-            ->setMethods(['invalidateCache'])
+            ->onlyMethods(['invalidateCache'])
             ->setConstructorArgs([$settingsManager, $this->getMockBuilder(CacheItemPoolInterface::class)->getMock(), 4711])
             ->getMock();
 
-        // Clear the cache
-        $cachedSettingsManager->expects($this->at(0))
+        $cachedSettingsManager->expects($this->exactly(2))
             ->method('invalidateCache')
-            ->with($this->equalTo($name), $this->equalTo($owner))
-            ->willReturn(true);
-
-        // Clear all cache for this owner
-        $cachedSettingsManager->expects($this->at(1))
-            ->method('invalidateCache')
-            ->with($this->equalTo(null), $this->equalTo($owner))
+            ->withConsecutive(
```

Comment:
->at() and ->withConsecutive() are both not supported in PhpUnit 10 ;(
Maybe there is a trait to solve this. Or later in a seperate PR